### PR TITLE
Add stats engine

### PR DESCRIPTION
**Pull request by cvolawless**
Add stats engine endpoint to the client + documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -360,6 +360,45 @@ See [the Optimizely REST API documentation](http://developers.optimizely.com/res
 # }
 ```
 
+###Get Experiment Results (Stats Engine)
+To list stats engine results, call stats on the associated `Experiment` object.
```
Comment (jgaul) [inline README.md]: nit: should be capital "Stats Engine". would also be great if this was a link to the Stats Engine documentation: https://help.optimizely.com/hc/en-us/articles/200039895-Stats-Engine-How-Optimizely-calculates-results-to-enable-business-decisions

**Comment by jgaul** [comment]
@cvolawless -- one nitpick, then LGTM! thank you!

**Comment by cvolawless** [comment]
Docs updated! Thanks @jgaul.

**Comment by jgaul** [comment]
thanks again, @cvolawless !

**Pull request merged**